pkg/repositories/repositories.go

```diff
@@ -57,6 +57,44 @@ func NewConfig(filename string) (Config, error) {
 	return repoConfig, nil
 }
 
+// If our first config and our second config share a repository,
+// we set the afterVersion of the first to its original version, and the
+// version of the first to the version of the second.
+func CombineConfigs(currentConfig *Config, newConfig *Config) {
```

Comment:
"current"/"new" aren't clearly enough delineated. Maybe "previousConfig"/"newConfig" or something similar would be clearer?

Reply:
It's unclear where is the output of this method. It may be useful to maybe either return a config or more clearly name the parameters.

Reply:
We're modifying the current config in-place, I think I know how to improve the description, how would you recommend renaming currentConfig? I'm not sure previousConfig is clear enough, since we're keeping it for the most part, just modifying it.